test.js

```diff
@@ -1,374 +1,361 @@
 'use strict';
 const path = require('path');
-const assert = require('assert');
+const test = require('ava');
 const gutil = require('gulp-util');
 const rev = require('./');
 
-it('should rev files', cb => {
+function createFile({path, revOrigPath, revOrigBase, origName, revName, cwd, base, contents = ''}) {
+	const file = new gutil.File({
+		path,
+		cwd,
+		base,
+		contents: Buffer.from(contents)
+	});
+	file.revOrigPath = revOrigPath;
+	file.revOrigBase = revOrigBase;
+	file.origName = origName;
+	file.revName = revName;
+
+	return file;
+}
+
+test.cb('should rev files', t => {
 	const stream = rev();
 
 	stream.on('data', file => {
```

Comment:
Use https://github.com/sindresorhus/p-event here and you don't need to use `test.cb()` and callback, but can take advantage of async functions. Example: https://github.com/sindresorhus/gulp-debug/blob/4db5871594742a346d17aa9b34f43c87d4e54934/test.js#L42-L44